.github/workflow-templates/cargo-build/action.yml

```diff
@@ -73,5 +73,5 @@ runs:
       run: |
         mkdir -p binaries
         cp target/release/tanssi-node binaries/tanssi-node;
-        cp target/release/container-chain-template-simple-node binaries/container-chain-template-simple-node;
-        cp target/release/container-chain-template-frontier-node binaries/container-chain-template-frontier-node;
+        cp target/release/container-chain-simple-node binaries/container-chain-simple-node;
```

Comment:
This file is in the "workflow-templates" folder, let's rename it to "workflow-containers"?

Reply:
Actually this mean 2 very different things. Workflow-templates only refer to "templates for workflows to be run in the CI". That is the reason why now the coverage does not work. Let's stay with `workflow-template`